sdk/loadtestservice/Azure.Developer.LoadTesting/src/Azure.Developer.LoadTesting.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.Text.Json" VersionOverride="6.0.6" />
```

Comment:
Why do we need to override this version? In general we should only do this in exception cases.

Reply:
@weshaggard, actually we wanted to use System.Text.Json.Nodes which is not available in the default version, in our preview release we won't be using it, so we removed `VersionOverride` for now.

Reply:
Talked to @christothes that confirmed I can resolve this conversation